src/view/components/fieldDescriptionAndError.jsx

```diff
@@ -34,8 +34,6 @@ const FieldDescriptionAndError = ({
   } else if (description) {
     messageClassName = "FieldDescriptionAndError-description";
     message = description;
-  } else {
-    return children;
```

Comment:
You may wonder why I changed this. Until I configured Formik to _not_ validate on change events, validation was happening on every change (keystroke). What I noticed is that if I typed a character that caused the field to go from valid to invalid (or vice-versa), the field would lose focus. That's because, in order to show the error message, we were taking the input out of its position in the DOM and then adding the div wrapper (see below). The fix is to not move the input around in the DOM while the user is typing, which is what this change fixes.

Now, since I made this change, I've turned off validation on change events, so moving the input in the DOM when validity changes is not a problem we'll run into, but I left this change in place in case we change our mind on when we want to validate.